src/main/java/pl/marcin/project/AppRunnerConfig.java

```diff
@@ -0,0 +1,27 @@
+package pl.marcin.project;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.ApplicationArguments;
+import org.springframework.boot.ApplicationRunner;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@RequiredArgsConstructor
```

Comment:
do we need this annotation? We actually create `appRunner` inside our `run` method so the generated constructor would be empty anyway.. ;)

Reply:
done